solvcon/parcel/gas/inout.py

```diff
@@ -65,12 +65,18 @@ def __init__(self, cse, show_bclist=False, perffn=None, **kw):
         super(MeshInfoHook, self).__init__(cse, **kw)
 
     def preloop(self):
+        flag_generic_bc = False
```

Comment:
Thanks.  It's a good idea to use a tool for checking the formatting.  Perhaps we should add that to travis.  There are many such tools and I don't know if flake8 is the best one.  I know there are a lot of legacy code in SOLVCON that will not pass the checks.  We need a proper suppression file.

Each time you re-submit the PR, the my comments are gone in the newly updated changesets.  I am not sure why it happens.